.travis.yml

```diff
@@ -3,18 +3,20 @@ language: php
 sudo: false
 
 php:
-  - 5.4
```

Comment:
Guzzle 6 requires >= 5.5, I can add an exclusion for those requirements if that sounds good.